### PR TITLE
Feature 77/add new apartment

### PR DESCRIPTION
**Pull request by rostovskiy88**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/utils/validators.ts [hunk 1]
```diff
@@ -60,3 +51,26 @@ export const adjoiningAreaValidator = helpers.withMessage('В назві маю
...
+export const addressValidator = helpers.withMessage(`${ukrLangMessage}`, ukranianLan);
+
+export const houseNumAndHouseBlockValidator = helpers.withMessage(
+  `${ukrLangMessage} або цифри від 0 до 9`,
+  validHouseNumAndHouseBlock
+);
+const validZipCode = helpers.regex(/^\d{5}$/);
+export const zipCodeValidator = helpers.withMessage('В назві мають бути 5 цифр від 0 до 9', validZipCode);
+
+const validPhoneNumber = helpers.regex(/^\+380\d{3}\d{2}\d{2}\d{2}$/);
```
Comment (aleonovdementiev) [inline src/utils/validators.ts:64]: Why do you changed validPhoneNumber validator? Won't it break existing logic in app?

File: src/views/ManageApartments.vue [hunk 2]
```diff
@@ -202,6 +219,14 @@ export default defineComponent({
       ];
     };
 
+    const displayApartmentModal = computed(() => {
+      return store.state.cooperationStore.displayModal;
```
Comment (aleonovdementiev) [inline src/views/ManageApartments.vue]: Do you use this store property in any other component?

File: src/utils/validators.ts (hunk 1 above)
Comment (rostovskiy88) [reply src/utils/validators.ts:64]: fixed

File: src/views/ManageApartments.vue (hunk 2 above)
Comment (rostovskiy88) [reply src/views/ManageApartments.vue]: fixed

**Review by aleonovdementiev — APPROVED** [review]

**Review by akononenko2375 — APPROVED** [review]

**Pull request merged**